src/query/storage/promremote/storage_test.go

```diff
@@ -176,10 +177,10 @@ func TestDataRace(t *testing.T) {
 				Name:  []byte("new_tag_name"),
 				Value: []byte("new_tag_value"),
 			},
-			{
-				Name:  []byte("new_tag_name2"),
-				Value: []byte("new_tag_value2"),
-			}},
+				{
+					Name:  []byte("new_tag_name2"),
+					Value: []byte("new_tag_value2"),
+				}},
```

Comment:
This reformatting is weird. I used golang's own formater.
<img width="612" alt="image" src="https://github.com/databricks/m3/assets/129140017/20cc7f89-22f3-44be-9043-ebffd35035b1">

Reply:
emm, i didn't do anything, my IDE auto format it, let me check